source/extensions/filters/http/ext_proc/client_impl.cc

```diff
@@ -0,0 +1,64 @@
+#include "extensions/filters/http/ext_proc/client_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+static constexpr char kExternalMethod[] =
+    "envoy.service.ext_proc.v3alpha.ExternalProcessor.Process";
+
+ExternalProcessorClientImpl::ExternalProcessorClientImpl(
+    Grpc::AsyncClientManager& client_manager,
+    const envoy::config::core::v3::GrpcService& grpc_service, Stats::Scope& scope) {
+  factory_ = client_manager.factoryForGrpcService(grpc_service, scope, true);
+}
+
+ExternalProcessorStreamPtr
+ExternalProcessorClientImpl::start(ExternalProcessorCallbacks& callbacks,
+                                   const std::chrono::milliseconds& timeout) {
+  Grpc::AsyncClient<ProcessingRequest, ProcessingResponse> grpcClient(factory_->create());
+  return std::make_unique<ExternalProcessorStreamImpl>(std::move(grpcClient), callbacks, timeout);
+}
+
+ExternalProcessorStreamImpl::ExternalProcessorStreamImpl(
+    Grpc::AsyncClient<ProcessingRequest, ProcessingResponse>&& client,
+    ExternalProcessorCallbacks& callbacks, const std::chrono::milliseconds& timeout)
+    : callbacks_(callbacks) {
+  client_ = std::move(client);
+  auto descriptor = Protobuf::DescriptorPool::generated_pool()->FindMethodByName(kExternalMethod);
+  assert(descriptor != nullptr);
```

Comment:
No need for an assert here, since it will crash immediately if not true. Also, we don't use lower case `assert` in Envoy, prefer `ASSERT`, `RELEASE_ASSERT` or `ENVOY_BUG.

Reply:
Ack. No assert needed for now then.